Locate config file (#4594)
influxdata/docs-v2: content/influxdb/v2.5/reference/cli/influx/config/create.md

---

The `influx config create` command creates a InfluxDB connection configuration

> and stores it as a hidden file in your home directory (`~/.influxdbv2/configs`).

Reviewer: ```suggestion
and stores it in a local file:

| OS/Platform                  | CLI config file path                |
| :--------------------------- | :---------------------------------- |
| macOS                        | `~/.influxdbv2/configs`             |
| Linux (installed as binary)  | `~/.influxdbv2/configs`             |
| Linux (installed as service) | `~/var/lib/influxdb/configs`        |
| Windows                      | `%USERPROFILE%\.influxdbv2\configs` |
| Docker (DockerHub)           | `/etc/influxdb2/configs`            |
| Docker (Quay.io)             | `/root/.influxdbv2/configs`         |
| Kubernetes                   | `/etc/influxdb2/configs`            |

```

> To view config files after creating them, use [influx config list](/influxdb/v2.5/reference/cli/influx/config/list/).

Reviewer: ```suggestion
To view CLI connection configurations after creating them, use [influx config list](/influxdb/v2.5/reference/cli/influx/config/list/).
```

{{% note %}}

> **Note:** If you create multiple config profiles (for example, separate admin and user configs), use [influx config set](/influxdb/v2.5/reference/cli/influx/config/set/) to switch to the config profile you want to use.

Reviewer: ```suggestion
**Note:** If you create multiple connection configurations (for example, separate admin and user configurations), use [`influx config <config-name>`](/influxdb/v2.5/reference/cli/influx/config/) to switch to the configuration you want to use.
```
